### PR TITLE
Log on bundle uploads

### PR DESCRIPTION
**Pull request by danihodovic**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] Automatic tests added/update
- [ ] Automatic tests pass
- [x] Software runs (locally or hosted)
- [x] Sanity check passed
- [x] Self-review passed

Currently we are in a state of failing bundle uploads without being aware of it.

### CODE REVIEW

**Comment by danihodovic** [comment]
Rebase on #322

File: src/services/data_model_engine.js [hunk 1]
```diff
@@ -217,8 +236,18 @@ export default class DataModelEngine {
         await this.entityRepository.storeBundleProofMetadata(waitingBundle.bundleId, blockNumber, timestamp, transactionHash);
         await this.bundleRepository.storeBundleProofMetadata(waitingBundle.bundleId, blockNumber, timestamp, transactionHash);
         summary.ok[waitingBundle.bundleId] = {uploadResult};
+        this.logger.info({
```
Comment (spherefoundry) [inline src/services/data_model_engine.js]: I don't like that the DataModelEngine is now aware of the logger. We managed to avoid this until now, and I would like to keep it this way. As an alternative I would propose the `uploadAcceptedBundleCandidate` method to accept a optional callback that can be used to report progress. It will be also easier to test (which you should also do)
